backend/users/serializers.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth.models import User
+from django.contrib.auth.password_validation import validate_password
```

Comment:
We are creating custom user model in the other branch. In order to make it work with the new user model you have to use `get_user_model` instead.